examples/test_examples.py

```diff
@@ -76,6 +79,41 @@ def test_run_glue(self):
             for value in result.values():
                 self.assertGreaterEqual(value, 0.75)
 
+    def test_run_pl_glue(self):
+        stream_handler = logging.StreamHandler(sys.stdout)
+        logger.addHandler(stream_handler)
+
+        testargs = """
+            run_pl_glue.py
+            --model_name_or_path bert-base-cased
+            --data_dir ./tests/fixtures/tests_samples/MRPC/
+            --task mrpc
+            --do_train
+            --do_predict
+            --output_dir ./tests/fixtures/tests_samples/temp_dir
+            --train_batch_size=32
+            --learning_rate=1e-4
+            --num_train_epochs=1
+            --seed=42
+            --max_seq_length=128
+            """.split()
+
+        if torch.cuda.is_available():
+            testargs += ["--fp16", "--gpus=1"]
+
+        with patch.object(sys, "argv", testargs):
+            result = run_pl_glue.main()
+            # for now just testing that the script can run to a completion
+            self.assertGreater(result["acc"], 0.25)
```

Comment:
Let's not forget to update this later, 0.25 accuracy is not impressive for a 2-label dataset.

Reply:
I'm getting acc/f1=1 on my machine. I just don't know why CI gets a much worser score with the same config. I'm all ears to what else I can try, as you can see from the commits I tried a bunch of hparams tweaking to no avail.

FWIW, @sshleifer was able to reproduce the low score on his machine.